finqwerty/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,7 @@
     <string name="pro1_qwerty_fin_1">FinQwerty F(x)tec Pro1, Finnish for physical US QWERTY</string>
     <string name="pro1_qwerty_hun_1">FinQwerty F(x)tec Pro1, Hungarian for physical US QWERTY</string>
     <string name="pro1_qwerty_nor_1">FinQwerty F(x)tec Pro1, Norwegian for physical US QWERTY</string>
+    <string name="pro1_qwerty_polprog_fndead">FinQwerty F(x)tec Pro1, Polish with Fn dead keys for physical US QWERTY</string>
```

Comment:
Is there a variant of the Polish programmer's layout that has *direct* (non-Fn/AltGr) dead keys?

If not, I think I'd prefer just dropping the "Fn dead keys", and making the id "pro1_qwerty_pol_prog" or "pro1_qwerty_polprog_1", unless you think the "with Fn dead keys" part is important.

Reply:
On another thought, I think I'd like an underscore between pol and prog in any case.

Reply:
As far as I'm aware the Polish Programmer's layout does not have dead keys at all, Fn or not. I based this layout on the usintl_fndead one, as I figured having optional dead keys is just an additional benefit for other european languages, which is why I named it that way. I don't have a preference for the naming, the various options you suggest make sense in their own way. Let me know what you'd like to see and I'll make the change.

Reply:
OK, let's make it 'pro1_qwerty_pol_prog' and remove "with Fn dead keys" from the description.

Reply:
Also, I think the description should mention programmer... maybe in parantheses like "Polish (programmer)" or in a similar way.